rfcs/01924_tls_attestation_ca.md

```diff
@@ -0,0 +1,169 @@
+# RFC#01924: Remote Attestation via TLS (Certificate Authority)
+
+## Objective
+
+Remotely attest applications in Trusted Execution Environments (TEEs) over the
+TLS protocol.
+
+## Background
+
+A lot of existing applications that would benefit from the remote attestation
+use **TLS protocol** to create secure channels with their clients. But
+integrating the remote attestation might bring up unnecessary complications in a
```

Comment:
The main problem is `completely rewriting the application connection logic`. For example, a legacy application used TLS to accept client connections, but now it would require something like EKEP instead etc.
Though I haven't found a dedicated report that describes problems of upgrading legacy applications to remote attestation.